app/models/katello/content_view.rb

```diff
@@ -469,7 +469,7 @@ def check_remove_from_environment!(env)
       }
 
       dependencies.each do |key, name|
-        if (models = self.association(key).scoped.in_environment(env)).any?
+        if (models = self.association(key).scope.in_environment(env)).any?
```

Comment:
In other places, `where(nil)` was used to replaced scoped, whereas here it was replaced with `scope`. What is the difference?